spaceship/spec/tunes/fixtures/iap_create_recurring.json

```diff
@@ -0,0 +1 @@
+{"sectionErrorKeys":[],"sectionInfoKeys":[],"sectionWarningKeys":[],"value":null,"id":null,"adamId":null,"appAdamIds":null,"familyId":null,"addOnType":"recurring","isNewsSubscription":false,"isReplaced":false,"replacementAdamId":null,"referenceName":{"value":"localizeddemo"},"productId":{"value":"x.a.a.b.b.c.d.x.y.z"},"clearedForSale":{"value":true},"pricingDurationType":{"value":null,"isEditable":true,"isRequired":false,"errorKeys":null},"pricingIntervals":[{"value":{"country":"WW","tierStem":"1","priceTierEndDate":null,"priceTierEffectiveDate":null}}],"ungrandfatheredIntervals":null,"freeTrialDurationType":{"value":null,"isEditable":true,"isRequired":false,"errorKeys":null},"bonusPeriodDurationType":{"value":null,"isEditable":true,"isRequired":false,"errorKeys":null},"versions":[{"id":null,"details":{"value":[{"value":{"description":{"value":"Description has at least 10 characters"},"name":{"value":"test name2"},"localeCode":"en-US"}},{"value":{"description":{"value":"German has at least 10 characters"},"name":{"value":"test name german2"},"localeCode":"de-DE"}}],"isEditable":true,"isRequired":false,"errorKeys":null},"contentHosting":null,"contentHostingData":null,"reviewNotes":{"value":"Some Review Notes here bla bla bla"},"reviewScreenshot":{"value":{"assetToken":null,"sortOrder":null,"type":null,"size":null,"width":null,"height":null,"checksum":null,"url":null,"thumbNailUrl":null,"originalFileName":null},"isEditable":true,"isRequired":false,"errorKeys":null},"merch":{"images":[{"id":null,"image":{"value":{"assetToken":null,"originalFileName":null,"width":null,"height":null,"checksum":null},"isEditable":true,"isRequired":false,"errorKeys":null},"status":null}],"showByDefault":true,"isActive":false},"status":null,"canBeSubmitted":false}],"missingRequiredPrivacyPolicyData":false,"missingRequiredFamilyDetail":false}
```

Comment:
@luongm unfortunately no, stubbing of an HTTP request is done by matching the string in this file to the one that the test sends. If there was a match the stubbed response is returned. Otherwise the test fails because it is not allowed to send requests in tests.